packages/server/scripts/docs/toSwagger.js

```diff
@@ -0,0 +1,320 @@
+let _ = require("lodash")
+let { pathToRegexp } = require("path-to-regexp")
+
+/********************************************************
+ * Based on: https://github.com/fsbahman/apidoc-swagger *
```

Comment:
Probably a good idea just copying this, considering how out of date all the swagger generator libs for apidoc are! Good starting point for changes we need to deal with going forward

Reply:
Its MIT licensed and I thought this method would give us way more control when we come to getting the swagger definitions fully formed, means we can alter the generated API Doc JSON before converting it which should be a fair bit easier to do.